### PR TITLE
velero backup tool

### PR DESCRIPTION
**Pull request by mmpetarpeshev**
This PR should be merged after the opensearch one : https://github.com/aeternity/terraform-aws-apps/pull/5 !

### CODE REVIEW

File: velero-backup-policy.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+            "Effect": "Allow",
+            "Action": [
+                "ec2:DescribeVolumes",
+                "ec2:DescribeSnapshots",
+                "ec2:CreateTags",
+                "ec2:CreateVolume",
+                "ec2:CreateSnapshot",
+                "ec2:DeleteSnapshot"
+            ],
+            "Resource": "*"
```
Comment (dincho) [inline velero-backup-policy.json:14]: We should limit the access to the corresponding environment/cluster.

File: velero-backup-bucket.tf [hunk 2]
```diff
@@ -0,0 +1,10 @@
+module "s3_bucket_velero_backup" {
+  source = "terraform-aws-modules/s3-bucket/aws"
+
+  bucket = "aeternity-velero-backup-${local.env_human}"
+  acl    = "private"
+  
+  versioning = {
+    enabled = false
```
Comment (dincho) [inline velero-backup-bucket.tf]: Enabling backup versioning as a layer of protection sounds reasonable to me, why not?

File: velero-backup-bucket.tf [hunk 3]
```diff
@@ -0,0 +1,10 @@
+module "s3_bucket_velero_backup" {
+  source = "terraform-aws-modules/s3-bucket/aws"
```
Comment (dincho) [inline velero-backup-bucket.tf:2]: please set a module version

**Review by dincho — COMMENTED** [review]
one issue, couple of comments/questions

File: velero-backup-bucket.tf (hunk 3 above)
Comment (mmpetarpeshev) [reply velero-backup-bucket.tf:2]: Done

File: velero-backup-policy.json (hunk 1 above)
Comment (mmpetarpeshev) [reply velero-backup-policy.json:14]: Not quite sure what should be there , I followed the official documentation.I suppose the velero role arn ? Will give it try to see the result. 
In that case I suppose we should do the same for lb policy? https://github.com/aeternity/terraform-aws-apps/blob/master/lb-controller-iam-policy.json

Comment (mmpetarpeshev) [reply velero-backup-policy.json:14]: Actually this policy is attached to the worker nodes , so nothing meaningful could be added except * , if you have something in mind let me know and I will change it.

Comment (dincho) [reply velero-backup-policy.json:14]: yes, the lb policy is not restrictive enough but that doesn't mean we should add more unrestricted policies. https://github.com/aeternity/terraform-aws-apps/issues/8

**Review by dincho — APPROVED** [review]

**Pull request merged**